src/inspector_agent.cc

```diff
@@ -483,6 +483,9 @@ class NodeInspectorClient : public V8InspectorClient {
   }
 
   void maxAsyncCallStackDepthChanged(int depth) override {
+    if (waiting_for_sessions_disconnect_) {  // No more async, event loops are shut down
```

Comment:
Nit: I am not sure what the comment exactly stands for. Maybe it's possible to rephrase it a tiny bit to make it even clearer?

Reply:
I did my best to reword the comment